### ISSUE TITLE
DateRangePicker post grid on clic on apply button

### ISSUE THREAD

**Issue by lsaintmaxent** [Nayjest/Grids#57]
Hey men,

I'm trying to improve the way that the dateRangePicker is working.

At the moment, you have to select the date range, then clic again on the filter, to press enter in order to filter the grid values.

Could it be more easy to use ? For exemple, can i force the grid to refresh when clic on the apply buton in custom range or another preset range (current month, this week,...)

Did you have tried this ?

**Comment by Nayjest** [comment]
Hi!
You can add "autosubmit" property to dateRangePicker grid component (setter & getter) and add there some javascript to submit form (only if 'autosubmit' turned on). I wil be glad to accept such improvement via pull request.

**Comment by lsaintmaxent** [comment]
Yeah, i'd love to help you improving your awesome grid module but i didn't get how to create a new js attrivute in the dateRangePicker component.

So the only change i've done to make it work, was to modify the DateRangePicker.js to add on the clickApply function and clickRange function this simple row : ; document.forms[0].submit();

I'm sorry, i really tried but i think it's a bit complicated to create the attribute management in grid configuration.

Sorry dude :/

**Comment by cawecoy** [comment]
I was looking for the same thing and just realized it was already solved https://github.com/Nayjest/Grids/pull/106

I guess this issue can be closed :)
